Sources/WordPressAuthenticator/Helpers/Authenticator/WordPressAuthenticator.swift

```diff
@@ -341,7 +341,6 @@ import WordPressKit
         }
     }
 
-    // Helper used by WPAuthTokenIssueSolver
     @objc
     public class func signinForWPCom(dotcomEmailAddress: String?, dotcomUsername: String?, onDismissed: ((_ cancelled: Bool) -> Void)? = nil) -> UIViewController {
```

Comment:
This made me wonder whether the method became unused, because there's no match in the diff. Alas, it's in use in  https://github.com/wordpress-mobile/WordPress-iOS/blob/05012641f2e381e90ea76913fb544a897a19c206/WordPress/Classes/ViewRelated/NUX/Helpers/WordPressAuthenticationManager.swift#L199-L205